mesonbuild/coredata.py

```diff
@@ -442,7 +442,7 @@ def __init__(self, options: argparse.Namespace, scratch_dir: str, meson_command:
             DependencyCache(self.options, MachineChoice.BUILD),
             DependencyCache(self.options, MachineChoice.HOST))
 
-        self.compiler_check_cache = OrderedDict()  # type: T.Dict[CompilerCheckCacheKey, compiler.CompileResult]
+        self.compiler_check_cache: T.Dict['CompilerCheckCacheKey', 'CompileResult'] = OrderedDict()
```

Comment:
This file goes down from 85 mypy errors to 84. Why didn't I just delete the broken stuff, lol...